### DISCUSSION TITLE
Low-Code Connector | Custom Streams

### DISCUSSION THREAD

**Discussion by Zawar92** [airbytehq/airbyte#36174] [Connector Questions]
Hi Team, 

We're in the process of developing a low-code connector that enables us to incorporate both custom streams and standard streams. This functionality mirrors what the [Google Analytics V4](https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors/source-google-analytics-v4) connector achieves using Python CDK.

Our approach involves employing a low-code methodology complemented by Python, utilizing CUSTOM_RETRIEVER. While we already have some standard streams in place, we also need to dynamically add custom streams during runtime, pulling configuration details as needed from `config.json`. How can we accomplish this?

Thank You.

**Comment by marcosmarxm** [comment]
@natikgadzhi any chance someone from extensibility can help? @Zawar92 at the moment there is some work being done to support custom schemas in the low-code which could collaborate to your custom connector.

**Reply by Zawar92** [reply]
Thanks @marcosmarxm, for the prompt reply. 
Do you have a timeline, for when this could be implemented?

[a comment by an automated account is not shown]
